README.md

```diff
@@ -1 +1,11 @@
-# IV
\ No newline at end of file
+# MyBusinessAPI
+
+## Descripción del problema
+
+Dado un caso real, en donde a una barbería normalmente acudía la gente sin previo aviso para esperar a poder ser atendida, tuvo que adaptarse a las nuevas normativas y habilitar un teléfono para la reserva de citas debido al COVID-19.
+
+Se plantea a partir de esta situación crear una API para la gestión de negocios. En esta se almacenaran datos del negocio gestionado, de los productos que pueda ofrecer para venta online, de los clientes, las citas de los mismos, de los pedidos que realicen, etc. 
```

Comment:
En esta fase sólo tenéis que plantear el problema, o sea que a partir de aquí lo podéis guardar para la siguiente.